app/javascript/components/public_health/PatientsTable.js

```diff
@@ -233,8 +236,10 @@ class PatientsTable extends React.Component {
 
     this.setState({ query, cancelToken, loading: true }, () => {
       axios
-        .get('/public_health/patients', {
-          params: { workflow: this.props.workflow, ...query },
+        .post('/public_health/patients', {
```

Comment:
Why did we make this a POST? We're not writing any values.

Reply:
It's a good idea to use POST instead of GET if you're dealing with huge parameters (which this can now grow to). Also better so potentially sensitive (search) data is not included in URL logs. Even just that latter reason in my mind makes this change worth it.